src/vivarium_cluster_tools/vipin/log_parser.py

```diff
@@ -10,82 +9,68 @@
 import requests
 
 
-class WorkerLog:
-
+class PerformanceLog:
     def __init__(self, file: Path):
         self.file = file
-        name = file.stem.split('.')  # TODO: validate name is <sge job id>.<task id>.log
-        self.sge_job_id = int(name[0])
-        self.task_id = int(name[1])
+        self.count = 0
 
     def get_summaries(self) -> dict:
-        """Get all performance summary log messages in WorkerLog"""
-        telemetry_pattern = re.compile(r'^\{\"host\".+\"job_number\".+\}$')
+        """Get all performance summary log messages in PerformanceLog"""
+        telemetry_pattern = re.compile(r'^{\"host\".+\"job_number\".+}$')
 
-        # Ideally we'd only need to look at the tail of the file, but because a worker can do multiple draws, we need
-        # to iterate over all the lines...
-        # TODO: either change worker to cut a new log for each draw and check tail reversed(list(log))
-        #  *OR* parallelize this
+        self.count = 0
```

Comment:
Count, on the other hand, looks like it should be a local variable.

Reply:
There isn't strong purpose for the count, it's unused. My original intent was to report the number of entries, but the count is redundant to, say, df.shape. I'll remove it.